packages/drivers/odsp-driver/src/getFileLink.ts

```diff
@@ -0,0 +1,204 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryLogger, ITelemetryProperties } from "@fluidframework/common-definitions";
+import { PromiseCache } from "@fluidframework/common-utils";
+import { canRetryOnError, getRetryDelayFromError } from "@fluidframework/driver-utils";
+import { PerformanceEvent } from "@fluidframework/telemetry-utils";
+import { getUrlAndHeadersWithAuth } from "./getUrlAndHeadersWithAuth";
+import { fetchHelper, getWithRetryForTokenRefresh } from "./odspUtils";
+import {
+    IdentityType,
+    OdspResourceTokenFetchOptions,
+    TokenFetcher,
+    tokenFromResponse,
+} from "./tokenFetch";
+
+async function getRequestIdsFromResponse(headers: Map<string, string>): Promise<ITelemetryProperties> {
```

Comment:
This function is moved in this PR: https://github.com/microsoft/FluidFramework/pull/5273
You need to pull the changes. I will approve after that.

Reply:
Thanks @jatgarg  for pointing it out. I merged and made appropriate changes.